esphome/components/esp32_hosted/__init__.py

```diff
@@ -16,50 +19,118 @@
 CODEOWNERS = ["@swoboda1337"]
 
 CONF_ACTIVE_HIGH = "active_high"
+
+CONF_SPI_FULL_DUPLEX = "spi-full-duplex"
```

Comment:
Constant CONF_SPI_FULL_DUPLEX does not match value spi-full-duplex
```suggestion
CONF_SPI_FULL_DUPLEX = "spi_full_duplex"
```